app/models/miq_request.rb

```diff
@@ -417,14 +417,35 @@ def task_check_on_execute
     raise _("approval is required for %{task}") % {:task => self.class::TASK_DESCRIPTION} unless approved?
   end
 
+  def miq_schedule
+    # HACK: this should be a real relation, but for now it's using a reserve_attribute for backport reasons
+    Reserve.where(:resource_type => "MiqSchedule").detect { |r| r.reserved == {:resource_id => id} }&.resource
+  end
+
   def execute
     task_check_on_execute
 
-    deliver_on = nil
     if get_option(:schedule_type) == "schedule"
-      deliver_on = get_option(:schedule_time).utc rescue nil
+      start_time = get_option(:schedule_time).utc rescue nil
```

Comment:
D'oh I meant to put [this](https://github.com/ManageIQ/manageiq/pull/17545/files#r193773386) here

Copying @gmcculloug 's comment (https://github.com/ManageIQ/manageiq/pull/17520#discussion_r193437017)
> I would recommend that we refactor this line. The classic UI sets get_option(:schedule_time) to a ActiveSupport::TimeWithZone object which supports the utc call.
>
> I suspect that if this is set from the API or some other means it would set the time as a string in which case the code will silently error and use nil. It is also a good opportunity to get rid of the inline rescue as well.

Reply:
Fixed by https://github.com/ManageIQ/manageiq/pull/17547